com.gitlab.KRypt0n_.an-anime-game-launcher.yml

```diff
@@ -0,0 +1,109 @@
+app-id: com.gitlab.KRypt0n_.an-anime-game-launcher
+runtime: org.gnome.Platform
+runtime-version: "41"
+sdk: org.gnome.Sdk
+separate-locales: false
+command: com.gitlab.KRypt0n_.an-anime-game-launcher
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11 # Not fallback-x11, because while the launcher supports Wayland, Wine doesn't (yet)
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=xdg-run/app/com.discordapp.Discord:create
+  - --talk-name=org.freedesktop.Notifications
+  - --device=all # Controllers
+
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /man
+  - /share/doc
+  - /share/gtk-doc
+  - /share/man
+  - /share/pkgconfig
+  - /share/vala
+  - "*.la"
+  - "*.a"
+
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+
+  - name: xdelta3
+    # Pretty sure autotools would've been the right option but that didn't work, so I used simple and mostly stole the commands from the Arch package
```

Comment:
you should still use autotools but you can add a "shell" source to do the reconfigure prior to it.